docs/README.md

```diff
@@ -1,35 +1,4 @@
-# TeslaMate Documentation
+## Documentation
```

Comment:
FYI it's `README.rst` (rather than this file), that's uses as the index page for https://teslamate.readthedocs.io/en/latest/